src/extension.ts

```diff
@@ -127,6 +138,73 @@ export function activate(context: ExtensionContext) {
         languageClient.sendRequest(ExecuteCommandRequest.type, request);
     }));
 
+    // Create the "open hub" command
+    // This is activated from the status bar (see below)
+    context.subscriptions.push(
+    commands.registerCommand("sourcery.hub.open", () => {
+
+      // Command is handled by the language server
```

Comment:
Change this comment here to something like - instruct the language server to start the server